### DISCUSSION TITLE
recast-navigation-three README should mention `await init()` needed

### DISCUSSION THREAD

**Discussion by rob-myers** [isaac-mason/recast-navigation-js#287] [Ideas]
Hey, thanks for the awesome work.

I couldn't get `@recast-navigation/three` to work without first importing `init` from `recast-navigation` and bootstrapping e.g.
```tsx
init().then(() => { ... })
```

If this is expected, it would help to include this step in the [README](https://github.com/isaac-mason/recast-navigation-js/blob/main/packages/recast-navigation-three/README.md#installation).

**Comment by isaac-mason** [comment]
Thanks! Yep this is a documentation miss. I've improved the README now: https://github.com/isaac-mason/recast-navigation-js/commit/4de57c04cd1cd3a75b082b763223650029ca1064

Let me know if you've got any other suggestions, much appreciated.
